src/plugins/intel_gpu/src/kernel_selector/kernels/concatenation/concatenation_kernel_ref.h

```diff
@@ -21,7 +21,10 @@ class ConcatenationKernelRef : public ConcatenationKernelBase {
     JitConstants GetJitConstants(const concatenation_params& params) const override;
     std::vector<FusedOpType> GetSupportedFusedOps() const override {
         return {
-            FusedOpType::REORDER
+            FusedOpType::REORDER,
+            FusedOpType::ACTIVATION,
+            FusedOpType::ELTWISE,
+            FusedOpType::QUANTIZE
```

Comment:
I don't see fused ops generator to be configured for concat. How is that supposed to work?